src/main/scala/cqrs/settings/Settings.scala

```diff
@@ -12,6 +12,7 @@ class Settings(system: ExtendedActorSystem) extends Extension {
   val config = system.settings.config getConfig ("cqrs")
 
   val maxOrderPrice: Double = config getDouble "maxOrderPrice"
+  val uninitializedOrderLifeTime: Duration = config getInt "uninitializedOrderLifeTime" seconds
```

Comment:
Typesafe config supports duration specification in config. This allows more flexibility in specifying the time.

```scala
import scala.concurrent.duration.{ Duration, MILLISECONDS }
val uninitializedOrderLifeTime: Duration = Duration(system.settings.config.getDuration("uninitializedOrderLifeTime", MILLISECONDS), MILLISECONDS)
```

and then in the config (see: https://github.com/typesafehub/config/blob/master/HOCON.md#duration-format): 
```
uninitializedOrderLifeTime = 30 seconds 
```